server/models/k8s_components_registration.go

```diff
@@ -64,12 +64,13 @@ func (cg *ComponentsRegistrationHelper) RegisterComponents(ctxs []*K8sContext, r
 						Component:     "core",
 						ComponentName: "Kubernetes",
 						EventType:     meshes.EventType_INFO,
+						Details:       "Registration of " + ctxName + " components started for contextID: " + ctxID,
 						Summary:       "Registration of " + ctxName + " components started for contextID: " + ctxID,
```

Comment:
```suggestion
						Details:       "MeshModel component registration has started for the Kubernetes context '" + ctxName + "' with context ID of " + ctxID,
						Summary:       "Component registration for Kubernetes context '" + ctxName + "' started.",
```

Making summary concise and details informational.